README.md

```diff
@@ -53,18 +53,17 @@ chmod +x minio
 ```
 
 ### Snap
-You can install the latest `minio` [snap](https://snapcraft.io), and help testing the most recent changes of the master branch in [all the supported Linux distros](https://snapcraft.io/docs/core/install) with:
+You can install the latest `minio` [snap](https://snapcraft.io).
 
 ```sh
 sudo snap install minio --edge
 ```
 
-Every time a new version of `minio` is pushed to the store, you will get it updated automatically.
-
-You will need to allow the minio snap to observe mounts in the system:
+Every time a new version of `minio` is pushed to the store, you will get it updated automatically. You will need to allow the minio snap to observe mounts in the system:
 
 ```sh
 sudo snap connect minio:mount-observe
+sudo snap run minio server /data
```

Comment:
On Ubuntu 17.10 I get the below error

```
sudo snap run minio server /data
minio: relocation error: /snap/minio/596/lib/x86_64-linux-gnu/libpthread.so.0: symbol __mmap, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```

Reply:
Looks like a local issue on your end @nitisht i can run fine on 17.10